src/main/java/com/robertx22/age_of_exile/database/data/talent_tree/parser/TalentGrid.java

```diff
@@ -60,21 +61,39 @@ public void loadIntoTree() {
 
         Watch fast2 = new Watch().min(1);
         // this can take a long time if the "hasPath" checks aren't minimized
```

Comment:
why are you optimizing this part? This only happens once on world load

Reply:
yeah I know, but this is also super slow on my PC so I add it on my branch, if it's unnecessary for the public build I can remove it.